### PR TITLE
Selfie: Use fused location

### PR DESCRIPTION
**Pull request by luca020400**
May avoid locking satellites

Signed-off-by: Luca Stefani <luca.stefani.ge1@gmail.com>
Change-Id: I74a142b2969637dde7eb677165a7adc5cc422c9f

### CODE REVIEW

**Review by luk1337 — CHANGES_REQUESTED** [review]
Breaks location on my device...

**Comment by luca020400** [comment]
This is currently broken as FusedLocationProvider crashes if either network or GPS provider don't exist.

That's stupid on AOSP side...

File: app/src/main/java/org/lineageos/selfie/MainActivity.kt [hunk 1]
```diff
@@ -51,6 +52,10 @@ import androidx.camera.view.video.OnVideoSavedCallback
 import androidx.camera.view.video.OutputFileResults
 import androidx.core.app.ActivityCompat
 import androidx.core.content.ContextCompat
+import androidx.core.location.LocationListenerCompat
+import androidx.core.location.LocationManagerCompat
+import androidx.core.location.LocationRequestCompat
+import androidx.core.location.LocationRequestCompat.QUALITY_BALANCED_POWER_ACCURACY
```
Comment (SebaUbuntu) [inline app/src/main/java/org/lineageos/selfie/MainActivity.kt:58]: Use LocationRequestCompat.QUALITY_BALANCED_POWER_ACCURACY instead

Comment (luca020400) [reply app/src/main/java/org/lineageos/selfie/MainActivity.kt:58]: I'd like to fix fused location first lol

**Pull request closed without merging**